src/frontend/package.json

```diff
@@ -31,15 +27,8 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "start": "npm run develop",
     "serve": "gatsby serve",
-    "clean": "gatsby clean"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
-  },
-  "bugs": {
-    "url": "https://github.com/gatsbyjs/gatsby/issues"
+    "clean": "gatsby clean",
+    "compile-sass": "node-sass src/sass/main.scss src/css/style.css -w"
```

Comment:
I think @miggs125 removed this line when he removed SASS.  It's safe to remove it here.

Reply:
@humphd line 32?

Reply:
Yes, just line 32, sorry for not being clear.  Line 31 is good, leave it in.